Makefile

```diff
@@ -5,7 +5,7 @@ ifndef GOBIN
 endif
 
 build: prepare
-	$(GOBIN)/godep go build -o telegraf -ldflags \
+	$(GOBIN)/godep go build -race -o telegraf -ldflags \
```

Comment:
OK.

Be aware that enabling race-detection is not free. Your code will run significantly slower.

https://golang.org/doc/articles/race_detector.html#Runtime_Overheads